packages/core/src/api/meta/initilialize.test.ts

```diff
@@ -1,6 +1,7 @@
-import { initializeFaro } from '@grafana/faro-core';
 import { mockConfig } from '@grafana/faro-core/src/testUtils';
 
+import { initializeFaro } from '../../initialize';
+
```

Comment:
why was this change necessary?

Reply:
Honestly I am not sure as I was trying things until the project built 😂 I will remove it and see if it is not required